src/ChainSafe.Gaming.Lootboxes.Chainlink/LootboxServiceConfig.cs

```diff
@@ -9,12 +12,47 @@ public class LootboxServiceConfig
         /// <summary>
         /// Gets or sets the Ethereum address of the smart contract that the LootboxService interacts with.
         /// </summary>
-        public string? ContractAddress { get; set; }
+        public string? LootboxAddress { get; set; }
```

Comment:
Why do we call the smart contract address property `LootboxAddress` now, if it's still the address of the contract?

Reply:
this is something else, incase you wanted to scan the balance of another account instead of your own and you dont want to override the function (something Nikola asked for)